flow/util/template_filters.py

```diff
@@ -156,6 +156,16 @@ def calc_num_nodes(np, ppn=1, threshold=0, name=None):
     return check_utilization(nn, np, ppn, threshold, name)
 
 
+def print_warning(warning):
+    """Print warning message within jinja2 template
+
+    :param:
+        The warning message as a string
+    """
+    print(warning)
+    return ''
```

Comment:
I assume this is necessary to make jinja2 happy?

Reply:
you mean the return '' part? yes with that jinja2 will put an empty line on the output file when the function is called and will not break anything